src/kfs_lib/includes/kfs_common.h

```diff
@@ -1,5 +1,9 @@
 #ifndef KFS_COMMON
 #define KFS_COMMON
+#define COPYMODE        0644
+#define BUFFERSIZE      4096
+#define PATH_MAX_EXTENDED        8192   //extended version of PATH MAX from limits.h (doubled size )
```

Comment:
it's not good to use direct value? why not use MAX_PATH*2?

Reply:
fixed 399cd8a